src/V3Sched.cpp

```diff
@@ -226,7 +226,8 @@ EvalLoop createEvalLoop(
     AstNodeStmt* stmtps = nullptr;
 
     // Prof-exec section push
-    if (v3Global.opt.profExec()) stmtps = profExecSectionPush(flp, "loop " + tag);
+    if (v3Global.opt.profExec() && !v3Global.opt.hierChild())
+        stmtps = profExecSectionPush(flp, "loop " + tag);
```

Comment:
Ideally shouldn't we be accounting for the child's time separately?

Reply:
This will be helpful in the analysis, but it will require somehow 'scoping' execution times of those functions. For instance, hierarchical modules have `eval` functions as well as the main module which causes conflicts in `verilator_gantt` and timing mismatches (thus added an assert).

We could show `Section profile` for each hierarchical module separately, but this would make results for hier and non-hier runs different.

Reply:
I'd expect a hier report would have additional entries corresponding to the additional partitioning, so seems ok in theory. But would need to see what the report looks like, as I agree we don't want it to be confusing.

Reply:
Or could we just make a couple of reports?
1. One for the whole design - as implemented in this PR
2. Separate reports for each hier module

This way every report will have the same structure and will be comparable with non-hier runs.
Same would apply to `.vcd`s

Reply:
I really want a single report, as performance is best improved with global visibility/knowledge.